### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by cklidify** [citra-emu/citra#1205]
Citra was working perfectly fine this morning but when I just tried to launch it now I got a "has stopped working" message with this in the console:

http://i.imgur.com/hp6JuHG.png

I am on Windows 8.1 if that helps

**Comment by Krak-n** [comment]
Can you confirm that the file/folder it was looking for is there? It seems as if Windows is unable to find: 
"\AppData\local\temp\Rar$EXa0.248/user/config/"

I'm no Windows expert, but I my best guess is something there is missing.

**Comment by cklidify** [comment]
The folder exists, but no file

**Comment by cklidify** [comment]
I just tried it again and now there's no console message: http://prntscr.com/8rfpaz

**Comment by Krak-n** [comment]
Question: What version on Qt are you using?

I'm reading the Windows FAQ and it seems as if 5.3 has some issues (Install 5.3.1 or greater). Since we're not getting any further console output, that's my best guess. Other than confirming your Qt version, possibly try rebuilding and see if the crash reoccurs?

**Comment by cklidify** [comment]
2 things:
1. What's Qt? (Sorry if that's like a super nooby question)
2. It worked perfectly fine this morning so i have no idea why a version or something would change

**Comment by Krak-n** [comment]
Disclaimer: I'm a newbie just like you!

I hate to do this, but it's hard to explain Qt if you're new to it:

https://en.wikipedia.org/wiki/Qt_(software)

The first paragraph is a pretty good overview. Basically it's an API that allows you to create applications that run on multiple platforms without much hassle.

I'm also curious as to why it stopped working when it had been working earlier for you.  If you don't want to rebuild it, I'm sure Windows has some more debugging applications that could shed some more light on what's going on.

If you're not in the mood to debug it, I'd recommend rebuilding the application following their Windows guide verbatim: https://github.com/citra-emu/citra/wiki/Windows-Build

**Comment by cklidify** [comment]
Alright, thanks! I'm away from my pc atm so I'll give it a shot tomorrow! Thanks!

**Comment by Andrix44** [comment]
Did you extract it?

**Comment by jroweboy** [comment]
To expand on what @Andrix44 is saying, you are running the application directly from the archive (the .zip file) which extracts citra to a temp folder and runs it from there, instead you should extract it and run it from the wherever you extracted it. Extract the archive and then run citra-qt and see if you have the same problems.

@Krak-n It looks like @cklidify is running the program from the download provided at citra.org which has the QT libraries inside it. No need to download QT for developers :)

**Comment by cklidify** [comment]
@Andrix44 I only ran it from a temp folder to show the error, I got the same error when running it in a non-temp folder

**Comment by cklidify** [comment]
I just downloaded a newer version of citra and got this error: http://prntscr.com/8rpj74

**Comment by Andrix44** [comment]
@cklidify I don’t know, then. Did you try to redownload it? How many files are in the folder? Maybe the temp files get automatically deleted, so when you took the second screenshot even more files were missing.

**Comment by Andrix44** [comment]
As i know they are automatically generated. I’ll check that tomorrow.

**Comment by cklidify** [comment]
@Andrix44 That most recent screenshot wasn't from running in a zip. That was it extracted.

**Comment by Andrix44** [comment]
@cklidify I know. Try running it as an admin. Maybe it doesn’t have write access. If that doesn't work, then try to create them manually. If it still nothing, then I can send you those files tomorrow to see if that works.

**Comment by cklidify** [comment]
Now when I try running it (as admin or normally) I get a "has stopped working" message before anything can even have a chance to appear in the console

**Comment by Andrix44** [comment]
Probably because the config file is still missing.

**Comment by cklidify** [comment]
I don't know what file it's asking for.

**Comment by Andrix44** [comment]
Try this: http://wikisend.com/download/929652/user.rar

**Comment by cklidify** [comment]
I tried that and I got this error: http://prntscr.com/8rw983 and that file does indeed exist

**Comment by Andrix44** [comment]
Sorry, I can't help you then.

**Comment by cklidify** [comment]
:/ If anyone else could possibly help me out then that'd be great! Or @Andrix44 if you could send me your entire citra folder I could try that too

**Comment by Andrix44** [comment]
There you go: http://wikisend.com/download/697146/citra-latest-windows-amd64.rar

**Comment by Andrix44** [comment]
The result?

**Comment by cklidify** [comment]
Sorry, I was at school. But it still gives me this same error: http://prntscr.com/8s1svp

**Comment by Krak-n** [comment]
Check to confirm if qt-config.ini exists.

**Comment by cklidify** [comment]
it exists, and these are the contents:
[Controls]
pad_a=65
pad_b=83
pad_x=90
pad_y=88
pad_l=81
pad_r=87
pad_zl=49
pad_zr=50
pad_start=77
pad_select=78
pad_home=66
pad_dup=84
pad_ddown=71
pad_dleft=70
pad_dright=72
pad_sup=16777235
pad_sdown=16777237
pad_sleft=16777234
pad_sright=16777236
pad_cup=73
pad_cdown=75
pad_cleft=74
pad_cright=76

[Core]
frame_skip=0

[Renderer]
use_hw_renderer=true
use_shader_jit=true
bg_red=1
bg_green=1
bg_blue=1

[Data%20Storage]
use_virtual_sd=true

[System%20Region]
region_value=1

[Miscellaneous]
log_filter=*:Info

**Comment by Krak-n** [comment]
Accidently deleted my comment instead of editing it. But... That's expected. I read the message wrong, and it's trying to create a new file there? At this point I'm wondering if a Dev would he able to shine some more light on the situation.

**Comment by cklidify** [comment]
Oh alright, thanks for your help anyways @Krak-n

**Comment by archshift** [comment]
Unfortunately this isn't very helpful. Is it possible for you to run this under a debugger (Visual Studio) and report to us the specific location where it crashes?

Otherwise, I'll probably have to close this issue.

**Comment by smartkaartik** [comment]
I'm also getting the same problem..please help

**Comment by smartkaartik** [comment]
How to run it under  under a debugger???

**Comment by mailwl** [comment]
https://github.com/citra-emu/citra/wiki/Windows-Build

**Comment by deadair3210** [comment]
I know this issue is old, but I'm having the same issue and was unsure weather making a dup or just using this issue was correct.

I open Cirta-Qt
Console opens with three errors that say path exist
Citra crashes

![capture9](https://cloud.githubusercontent.com/assets/1053504/12053007/574a3b6c-aeda-11e5-9f16-3e64b704dbca.PNG)
![capture8](https://cloud.githubusercontent.com/assets/1053504/12053008/574a7e88-aeda-11e5-92b1-67d836a3079d.PNG)

**Comment by deadair3210** [comment]
Built citra as per the instructions, same result. Wa
![capture10](https://cloud.githubusercontent.com/assets/1053504/12055814/4a08704c-aef6-11e5-9464-02e8e6105248.PNG)
sn't sure if this would be useful or not but tried anyway

**Comment by yuriks** [comment]
This looks like the issue @Normatt was talking about where the gamelist settings don't have default values.

**Comment by deadair3210** [comment]
How would I test this? Would be glad to help

**Comment by deadair3210** [comment]
I get this from the terminal if I build on Linux. I'm guessing you are correct in that for some reason it looks like citra isnt either finding the right values or a file at alll. To me it looks like a variable isnt getting set because it doesnt list even a folder its looking in
![screenshot from 2015-12-30 20 01 45](https://cloud.githubusercontent.com/assets/1053504/12060233/a921e8e4-af30-11e5-9f52-5433208b4438.png)

**Comment by deadair3210** [comment]
Also, are you sure you mentioned the right person, looked at the account you said talked about this before and the account has had no activity

**Comment by yuriks** [comment]
Oops. @Normmatt. Github mobile doesn't have auto-completion. :P Anyway, to work around this, try to add entries for the two settings to your configuration file.

**Comment by deadair3210** [comment]
where do i add it in the config file, there isnt a specific section as far as i see

**Comment by deadair3210** [comment]
[Controls]
pad_a=65
pad_b=83
pad_x=90
pad_y=88
pad_l=81
pad_r=87
pad_zl=49
pad_zr=50
pad_start=77
pad_select=78
pad_home=66
pad_dup=84
pad_ddown=71
pad_dleft=70
pad_dright=72
pad_sup=16777235
pad_sdown=16777237
pad_sleft=16777234
pad_sright=16777236
pad_cup=73
pad_cdown=75
pad_cleft=74
pad_cright=76

[Core]
frame_skip=0

[Renderer]
use_hw_renderer=false
use_shader_jit=true
bg_red=1
bg_green=1
bg_blue=1

[Data%20Storage]
use_virtual_sd=true

[System%20Region]
region_value=1

[Miscellaneous]
log_filter=*:Info

[Debugging]
use_gdbstub=false
gdbstub_port=24689

**Comment by archshift** [comment]
I'm pretty sure @Normmatt told me on IRC that the issue wasn't fixed even with the PR, so reopening.

**Comment by Normmatt** [comment]
Yeah that didn't fix it for me.. It seems to be related to the UILayout setting geometryRenderWindow at least for me. Deleting that from the ini fixed the crash for me.

**Comment by yuriks** [comment]
Is this still an issue?

**Comment by 28mane** [comment]
Yes It's still an issue with me.
I am getting same issue what deadair3210 got on linux but on windows.

**Comment by ddelapaz** [comment]
I'm also encountering this issue... It ran fine the first time then all the sudden this is happeneing.

```
The thread 0x3070 has exited with code 0 (0x0).
Unhandled exception at 0x00007FF855128BA0 (Qt5Gui.dll) in citra-qt.exe: 0xC000041D: An unhandled exception was encountered during a user callback.

```


```
00007FF85B848B9C ret 

00007FF85B848B9D int 3 

00007FF85B848B9E int 3 

00007FF85B848B9F int 3 

00007FF85B848BA0 mov rax,qword ptr [rcx+8] //BREAKS HERE
00007FF85B848BA4 movups xmm0,xmmword ptr [rax+88h] 

00007FF85B848BAB mov rax,rdx 

00007FF85B848BAE movups xmmword ptr [rdx],xmm0 

00007FF85B848BB1 ret 

00007FF85B848BB2 int 3 

00007FF85B848BB3 int 3 

00007FF85B848BB4 int 3 

00007FF85B848BB5 int 3
```

**Comment by ddelapaz** [comment]
I managed to get the `citra-qt.exe` to run again. 

Head over to `C:\Users\{UsernameHere}\AppData\Roaming\Citra team`

Mine looked like so:

```
[General]
recentFiles=C:/Users/intru/Desktop/CITRA/Pokemon-X.3ds
romsPath=C:/Users/intru/Desktop/CITRA
singleWindowMode=true
displayTitleBars=true
firstStart=false
confirmClose=true

[UILayout]
geometry=@ByteArray(\x1\xd9\xd0\xcb\0\x2\0\0\0\0\a\xb5\0\0\x1\xac\0\0\f\xc4\0\0\x3\xee\0\0\a\xbd\0\0\x1\xcb\0\0\f\xbc\0\0\x3\xe6\0\0\0\x2\0\0\0\0\a\x80)
state=@ByteArray(\0\0\0\xff\0\0\0\0\xfd\0\0\0\x2\0\0\0\x1\0\0\0\0\0\0\0\0\xfc\x2\0\0\0\b\xfb\0\0\0\x18\0\x41\0R\0M\0R\0\x65\0g\0i\0s\0t\0\x65\0r\0s\0\0\0\0\0\xff\xff\xff\xff\0\0\0o\0\xff\xff\xff\xfb\0\0\0\x12\0\x43\0\x61\0l\0l\0S\0t\0\x61\0\x63\0k\0\0\0\0\0\xff\xff\xff\xff\0\0\0o\0\xff\xff\xff\xfb\0\0\0 \0G\0r\0\x61\0p\0h\0i\0\x63\0s\0\x44\0\x65\0\x62\0u\0g\0g\0\x65\0r\0\0\0\0\0\xff\xff\xff\xff\0\0\0_\0\xff\xff\xff\xfb\0\0\0\"\0P\0i\0\x63\0\x61\0 \0\x43\0o\0m\0m\0\x61\0n\0\x64\0 \0L\0i\0s\0t\0\0\0\0\0\xff\xff\xff\xff\0\0\0\x8c\0\xff\xff\xff\xfb\0\0\0*\0P\0i\0\x63\0\x61\0\x42\0r\0\x65\0\x61\0k\0P\0o\0i\0n\0t\0s\0W\0i\0\x64\0g\0\x65\0t\0\0\0\0\0\xff\xff\xff\xff\0\0\0\x8c\0\xff\xff\xff\xfb\0\0\0\x1e\0P\0i\0\x63\0\x61\0\x46\0r\0\x61\0m\0\x65\0\x62\0u\0\x66\0\x66\0\x65\0r\0\0\0\0\0\xff\xff\xff\xff\0\0\0\xd4\0\xff\xff\xff\xfb\0\0\0 \0P\0i\0\x63\0\x61\0V\0\x65\0r\0t\0\x65\0x\0S\0h\0\x61\0\x64\0\x65\0r\0\0\0\0\0\xff\xff\xff\xff\0\0\x1\x9a\0\xff\xff\xff\xfb\0\0\0\x12\0\x43\0i\0T\0r\0\x61\0\x63\0i\0n\0g\0\0\0\0\0\xff\xff\xff\xff\0\0\0?\0\xff\xff\xff\0\0\0\x3\0\0\0\0\0\0\0\0\xfc\x1\0\0\0\x2\xfb\0\0\0\x10\0P\0r\0o\0\x66\0i\0l\0\x65\0r\0\0\0\0\0\xff\xff\xff\xff\0\0\0Y\0\xff\xff\xff\xfb\0\0\0\x14\0\x44\0o\0\x63\0k\0W\0i\0\x64\0g\0\x65\0t\0\0\0\0\0\xff\xff\xff\xff\0\0\0\xca\0\xff\xff\xff\0\0\x5\0\0\0\x2\a\0\0\0\x4\0\0\0\x4\0\0\0\b\0\0\0\b\xfc\0\0\0\0)
geometryRenderWindow=@ByteArray(\x1\xd9\xd0\xcb\0\x2\0\0\0\0\0\0\0\0\0\0\0\0\x1\x8f\0\0\x1\xdf\0\0\0\0\0\0\0\0\0\0\x1\x8f\0\0\x1\xdf\0\0\0\0\0\0\0\0\a\x80)
microProfileDialogGeometry="@ByteArray(\x1\xd9\xd0\xcb\0\x2\0\0\0\0\x1\xc4\0\0\0\xb6\0\0\x5\xbb\0\0\x3\x34\0\0\x1\xcc\0\0\0\xd5\0\0\x5\xb3\0\0\x3,\0\0\0\0\0\0\0\0\a\x80)"
microProfileDialogVisible=false
gameListHeaderState=@ByteArray(\0\0\0\xff\0\0\0\0\0\0\0\x1\0\0\0\x1\0\0\0\x3\x1\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\x4\xec\0\0\0\x3\x1\x1\0\x1\0\0\0\0\0\0\0\0\0\0\0\0\x64\xff\xff\xff\xff\0\0\0\x81\0\0\0\0\0\0\0\x3\0\0\0\x64\0\0\0\x1\0\0\0\0\0\0\0\x64\0\0\0\x1\0\0\0\0\0\0\x4$\0\0\0\x1\0\0\0\0\0\0\x3\xe8\0)

[Shortcuts]
Disassembler\Set%20Breakpoint\KeySeq=F9
Disassembler\Set%20Breakpoint\Context=2
Disassembler\Start\Stop\KeySeq=F5
Disassembler\Start\Stop\Context=2
Disassembler\Step\KeySeq=F10
Disassembler\Step\Context=2
Disassembler\Step%20into\KeySeq=F11
Disassembler\Step%20into\Context=2
Main%20Window\Load%20File\KeySeq=Ctrl+O
Main%20Window\Load%20File\Context=1
Main%20Window\Start%20Emulation\KeySeq=
Main%20Window\Start%20Emulation\Context=1

```

I deleted the entire file and the emulator loaded up fine.

**Comment by Lord-Haji** [comment]
I deleted the entire file same error as http://prntscr.com/asfb6w

**Comment by MerryMage** [comment]
@Lord-Haji Not the same issue, the above issue is about crashing at startup.

@ddelapaz Could you replace your Citra.ini file in a recent version of citra with what you've pasted above and check if it crashes or not? I'm having trouble reproducing this issue.

**Comment by wwylele** [comment]
Hit by this issue today. The following line in `qt-config.ini` is causing crashing on startup. If I remove it from ini, everything is fine; if I add it to a new-created ini, citra crashes.

```
[UI]
UILayout\geometry=@ByteArray(\x1\xd9\xd0\xcb\0\x2\0\0\0\0\a\x89\0\0\0\n\0\0\f7\0\0\x3\xee\0\0\a\x91\0\0\0)\0\0\f/\0\0\x3\xe6\0\0\0\x1\0\0\0\0\a\x80)
```

I suspect that this is a bug in Qt library. The call stack goes deeply into Qt library and crashes (same as @deadair3210 's screenshots)

**Comment by colateralxo99** [comment]
Hello I am having similar issues with Citra, http://imgur.com/KddmZ85 this is what I get when i launch citra-qt, and when i launch the other citra application the command prompt appears for a split second and then closes. I have extracted the file to a folder on my desktop as instructed however i get this error. Can anyone help ?

**Comment by colateralxo99** [comment]
hello if anyone can respond please.. Id love some help.

**Comment by jroweboy** [comment]
@colateralxo99 please, only one post is good enough. the second one is just spam, and no one likes spam. Next up, why do you have the msvcp dll in the folder. Delete both msvcp140.dll and vcruntime.dll as they should be installed through this https://www.microsoft.com/en-us/download/details.aspx?id=48145 Once you have that installed, try it again.

**Comment by LittleWhite-tb** [comment]
Yes, I think you are running "citra". You should start citra_qt.exe :)

**Comment by colateralxo99** [comment]
ok I installed that, and i get the same error the command prompt appears but nothing happens

**Comment by Christianwidjaya** [comment]
guys, this is my problem
http://prnt.sc/bymr23
running latest citra (downloaded July 28th 2016) in admin access mode.
help?

**Comment by legoj15** [comment]
Christian, you're using what appears to be an illegally downloaded ROM. Those do not receive help because they go against Citra's rules. It also happens that the game you are trying to use does not currently work on Citra due to a regression.

**Comment by JayFoxRox** [comment]
@legoj15 nothing suggests that. Everybody can name their files however they wish. Unless it names a warez site or warez group it's probably just how people categorize their backups.

---

@Christianwidjaya 
@colateralxo99 

This is a bug tracker for developers of Citra.

Please use the [Citra Forum for questions and support](https://discuss.citra-emu.org/) instead.
Note that there is no support if you use pirated copies of the game or 3DS bios files.

Also, do not re-use existing issues for your own issues (= Don't hijack issues by asking random questions). Only post if you can contribute valuable information.

---

This seems to have been abused as a general-goto-thread for "my citra crashes". Most problems are addressed by #1944 probably.

Create a new issue (with actual backtraces) if the problem still persists.

I'll close and lock this now.
